lms/djangoapps/django_comment_client/forum/views.py

```diff
@@ -148,7 +147,7 @@ def get_threads(request, course, discussion_id=None, per_page=THREADS_PER_PAGE):
             group_id = None
     else:
         group_id = get_cohort_id(request.user, course.id)
-        if not group_id and get_cohorted_threads_privacy(course.id) == 'cohort-only':
```

Comment:
Yes. The possibility of being outside of a cohort in a cohorted course was considered a bug and fixed by adding the 'default cohort'.